exercises/variables/variables2.d

```diff
@@ -0,0 +1,8 @@
+/// Something is wrong with the test!
+/// Fix it without modifying the variable declaration.
+
+unittest
+{
+	int i; // DO NOT MODIFY THIS LINE
+	assert(i == 2);
```

Comment:
Since the objective is for users to understand D default initializes variables, I'd change the "rules" to lead users in that direction. My first instinct here would be to write a new line `i = 2;`, but if the comment was like this, I'd try to figure out the intended way:
```suggestion
	assert(i == 2); // Modify this line only.
```

Reply:
Fixed in 35110d874775bdc449554e9e49bbf2c334a8bd65